src/utils/client.js

```diff
@@ -3,13 +3,20 @@ import { initFormsPromise, initInfo, initState } from './init';
 import { encodeGetParams } from './primitives';
 
 // Convenience boolean for urls - manually change for testing
-const isLocal = false;
-export const API_URL = isLocal
-  ? 'http://localhost:8006/api/'
-  : 'https://api.feathery.io/api/';
-export const CDN_URL = isLocal
-  ? 'http://localhost:8006/api/'
-  : 'https://cdn.feathery.io/api/';
+const API_URL_OPTIONS = {
+  local: 'http://localhost:8006/api/',
+  staging: 'https://staging.feathery.io/api/',
+  production: 'https://api.feathery.io/api/'
+};
+
+const CDN_URL_OPTIONS = {
+  local: 'http://localhost:8006/api/',
+  staging: 'https://staging.feathery.io/api/',
+  production: 'https://cdn.feathery.io/api/'
+};
+
+export const API_URL = API_URL_OPTIONS.production;
```

Comment:
should `production` be a configurable variable? i.e.

const environment = 'production';

export const API_URL = API_URL_OPTIONS[environment[